patches/react-native-mmkv+2.5.1.patch

```diff
@@ -26,10 +26,10 @@ index 0c5fe95..9fdf560 100644
        // Store the Object as a serialized Value
        setString(JSON.stringify(v));
 diff --git a/node_modules/react-native-mmkv/lib/typescript/hooks.d.ts b/node_modules/react-native-mmkv/lib/typescript/hooks.d.ts
-index 54c6ae9..6d992a7 100644
+index b4a07cd..4a8856d 100644
 --- a/node_modules/react-native-mmkv/lib/typescript/hooks.d.ts
 +++ b/node_modules/react-native-mmkv/lib/typescript/hooks.d.ts
-@@ -53,7 +53,7 @@ export declare const useMMKVBoolean: (key: string, instance?: MMKV | undefined)
+@@ -64,7 +64,7 @@ export declare const useMMKVBuffer: (key: string, instance?: MMKV | undefined) =
```

Comment:
This changed because MMKV added useMMKVBuffer hook and the order of declarations changed and we used to have a patch that I wanted to leave in case removing would cause some regressions